src/qhttp/Server.cc

```diff
@@ -303,8 +307,12 @@ void Server::_readRequest(std::shared_ptr<ip::tcp::socket> socket)
                                 << "request body read failed: " << ec.message());
                             return;
                         }
-                        if (request->header["Content-Type"] == "application/x-www-form-urlencoded") {
-                            request->_parseBody();
+                        if ((request->header["Content-Type"] == "application/x-www-form-urlencoded")
```

Comment:
In reality, all existing applications in Qserv expect a different content-type:
```
application/json
```
Please, see this class https://github.com/lsst/qserv/blob/main/src/replica/HttpRequestBody.cc
It would be natural if `qhttp` would provide the built-in support for that. Though, it would mean making `qhttp` dependent on `nlohmann::json`.

Reply:
Perhaps in a separate ticket ... if you will find this suggestion reasonable in the context of `qhttp`?

Reply:
I like that idea!  It's a very general capability, so makes sense in the context of qhttp.  And `nlohmann::json` is a pretty light dependency.  Let's do this in a separate ticket.